core/src/syn/v1/stmt/define/table.rs

```diff
@@ -23,13 +29,15 @@ pub fn table(i: &str) -> IResult<&str, DefineTableStatement> {
 	let (i, name) = cut(ident)(i)?;
 	let (i, opts) = many0(table_opts)(i)?;
 	let (i, _) = expected(
-		"DROP, SCHEMALESS, SCHEMAFUL(L), VIEW, CHANGEFEED, PERMISSIONS, or COMMENT",
+		"TYPE, RELATION, DROP, SCHEMALESS, SCHEMAFUL(L), VIEW, CHANGEFEED, PERMISSIONS, or COMMENT",
 		ending::query,
 	)(i)?;
 	// Create the base statement
 	let mut res = DefineTableStatement {
 		name,
 		permissions: Permissions::none(),
+		// Default to ANY if not specified in the DEFINE statement
+		kind: TableType::Any,
```

Comment:
Shouldn't this be taken care of by the `..Default::default()` bit?

Reply:
yes this is left over from when some things didn't default to Any so I was being clear, but its redundant now, I'll get rid of it.